doc/rofi.1.markdown

```diff
@@ -627,35 +619,35 @@ Disable parsing of configuration. This runs rofi in *stock* mode.
 
 Disables the loading of plugins.
 
-To get a trace with (lots of) debug information set the following environment variable when executing **rofi**:
+To get a trace with (lots of) debug information, set the following environment variable when executing **rofi**:
 
 ```
 G_MESSAGES_DEBUG=all
 ```
 
-The trace can be filtered by only outputting the relevant domains, f.e.:
+The trace can be filtered by only outputting the relevant domains, for example:
 
 ```
 G_MESSAGES_DEBUG=Dialogs.DRun
 ```
 
-For more information on debugging see the [wiki](https://github.com/DaveDavenport/rofi/wiki/Debugging%20Rofi)
+For more information on debugging, see the [wiki](https://github.com/DaveDavenport/rofi/wiki/Debugging%20Rofi)
 
 ## PATTERN
 
-To launch commands (e.g. when using the ssh launcher) the user can enter the used command-line,
+To launch commands (e.g. when using the ssh launcher), the user can enter the used command-line,
```

Comment:
hmm that is very vague yes.. 
These are keyworks that can be used in several options, like `-run-command` and '-ssh-command'.

When launching applications (for example using the *run* mode) the command executed can be specified, e.g. using the `-run-command` configuration option. The following keywords in the command will be replaced at run-time by their value.  

Not sure if this is more clear.